packages/renderer/src/lib/window-control-buttons/ControlButtons.spec.ts

```diff
@@ -46,7 +46,7 @@ describe.each([{ platform: 'linux' }, { platform: 'win32' }])('Platform($platfor
   test('Expect minimize is called', async () => {
     render(ControlButtons, { platform });
 
-    const minimizeMock = vi.fn();
+    const minimizeMock = vi.fn().mockResolvedValue(undefined);
```

Comment:
not sure to follow why we have this change ?

Reply:
usually our functions are async and then error will be thrown to the svelte button

so probably better to add async tot the method + await rather catch